standard.mk

```diff
@@ -18,40 +18,37 @@ endif
 # Generate version and tag information from inputs
 COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-9]{40}$$"`..HEAD --count)
 CURRENT_COMMIT=$(shell git rev-parse --short=7 HEAD)
-OPERATOR_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${COMMIT_NUMBER}-${CURRENT_COMMIT}
+OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
 
-OPERATOR_IMAGE_URI=${IMAGE_REGISTRY}/${IMAGE_REPOSITORY}/${IMAGE_NAME}:v${OPERATOR_VERSION}
-OPERATOR_IMAGE_URI_LATEST=${IMAGE_REGISTRY}/${IMAGE_REPOSITORY}/${IMAGE_NAME}:latest
-OPERATOR_DOCKERFILE?=build/Dockerfile
+OPERATOR_IMAGE_URI=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):v$(OPERATOR_VERSION)
+OPERATOR_IMAGE_URI_LATEST=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME):latest
+OPERATOR_DOCKERFILE ?=build/Dockerfile
 
-BINFILE=build/_output/bin/${OPERATOR_NAME}
+BINFILE=build/_output/bin/$(OPERATOR_NAME)
 MAINPACKAGE=./cmd/manager
+unexport GOFLAGS
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
-GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
+GOBUILDFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
-CONTAINER_ENGINE=$(shell which podman 2>/dev/null || which docker 2>/dev/null)
+CONTAINER_ENGINE=$(shell command -v podman 2>/dev/null || command -v docker 2>/dev/null)
 
-TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
 # ex, -v
 TESTOPTS :=
 
 ALLOW_DIRTY_CHECKOUT?=false
 
-default: gobuild
```

Comment:
When I build with `make` locally it's running the "clean" target.  It don't expect it to, but it is.  I wonder if it's defaulting to the first defined target if it hasn't seen a "default" first?  The file `standard.mk` is loaded before "default" is defined in `Makefile`.  Need to add this back in for expected behavior.